packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableTokenImage.tsx

```diff
@@ -20,11 +22,29 @@ export const TransactionsTableTokenImage = ({
 
   const token = useMemo(() => {
     return allTokens.find(
-      t => t.symbol.toLowerCase() === tokenSymbol.toLowerCase()
+      t => t.symbol.toLowerCase() === tx.asset.toLowerCase()
     )
-  }, [allTokens, tokenSymbol])
+  }, [allTokens, tx.asset])
+
+  if (tx.assetType === AssetType.ETH) {
+    const orbitChain = orbitChains[tx.childChainId]
+
+    const nativeTokenLogoSrc =
+      orbitChain?.bridgeUiConfig.nativeTokenData?.logoUrl
+
+    if (nativeTokenLogoSrc) {
+      return (
+        // SafeImage is used for token logo, we don't know at buildtime where those images will be loaded from
```

Comment:
The comment doesn't seem valid anymore, we're not using `SafeImage`

Reply:
updated the comment